pkg/clusterlogging/const.go

```diff
@@ -0,0 +1,16 @@
+package clusterlogging
+
+const (
```

Comment:
Please remove some unused variables and move rest of them to _test file + make them private.